src/test/java/roomescape/admin/controller/AdminControllerTest.java

```diff
@@ -0,0 +1,174 @@
+package roomescape.admin.controller;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.annotation.Rollback;
+import roomescape.admin.dto.SaveReservationRequest;
+import roomescape.admin.dto.SaveReservationTimeRequest;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class AdminControllerTest {
+
+    SaveReservationTimeRequest given_예약시간_10_20 = new SaveReservationTimeRequest("10:20");
+    SaveReservationTimeRequest given_예약시간_13_20 = new SaveReservationTimeRequest("13:20");
+    SaveReservationRequest given_10_20분_예약 = new SaveReservationRequest("김민기","2024-01-10", 1L);
+
+
+    @Nested
+    @DisplayName("방탈출 예약은")
+    class Describe_Resevation{
+
+        @Test
+        @DisplayName("사전 예약 등록 후 예약할 수 있습니다")
+        @Rollback
+        void save_reservation(){
+            Describe_Reservation_Time reservationTime = new Describe_Reservation_Time();
+            reservationTime.save_reservation_time(given_예약시간_10_20);
+
+            var response = RestAssured
+                    .given().log().all()
+                    .body(given_10_20분_예약)
+                    .contentType(ContentType.JSON)
+                    .when().post("/reservations")
+                    .then().log().all().extract();
+
+            assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+            assertThat(response.jsonPath().getString("name")).isEqualTo("김민기");
+        }
+
+        @Test
+        @DisplayName("목록을 조회할 수 있습니다.")
+        @Rollback
+        void return_list() throws JsonProcessingException {
+            save_reservation();
+
+            var response = RestAssured
+                    .given().log().all()
+                    .contentType(ContentType.JSON)
+                    .when().get("/reservations")
+                    .then().log().all().extract();
+
+            // 응답을 String으로 추출합니다.
+            String jsonResponse = response.asString();
+
+            // ObjectMapper를 사용하여 JSON 응답을 List로 변환합니다.
+            ObjectMapper objectMapper = new ObjectMapper();
+            List<Object> reservations = objectMapper.readValue(jsonResponse, new TypeReference<List<Object>>() {});
+
+            assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+            assertThat(reservations.size()).isGreaterThan(0);
```

Comment:
`isPositive()` 를 사용할 수 있을 것 같아요 

```suggestion
            assertThat(reservations.size()).isPositive();
```